server/src/modules/data/usecases/getPilotageReformeStats/getPilotageReformeStats.schema.ts

```diff
@@ -33,7 +33,7 @@ export const getPilotageReformeStatsSchema = {
         regions: z.array(OptionSchema),
         diplomes: z.array(OptionSchema),
       }),
-      tauxTransformation: z.number(),
+      tauxTransfoCumule: z.number().optional(),
```

Comment:
```suggestion
      tauxTransformationCumule: z.number().optional(),
```

Pas forcément besoin de réduire le nom IMO